cal/cal/templates/home_logged_in.html

```diff
@@ -117,6 +117,8 @@ <h4 class="well">Categories - Last Week</h4>
             </tr>
           </tbody>
         </table>
+<!--       <nvd3 options="options" data="getFilteredCategories"></nvd3>-->
+<!--       <nvd3 options="options" data="categories | filter:category:include"></nvd3>-->
```

Comment:
This isn't working because the `expression` field needs to be a string, Object, or function. I recommend filtering using the Object. Looks like you're using the comparator field wrong too. Full specification in the docs [here](https://docs.angularjs.org/api/ng/filter/filter) (Look at 'arguments'). If you can't figure out the docs let me know!